docs/etcd/demo_cluster_setup.md

```diff
@@ -1,14 +1,42 @@
 # Set up a demo etcd installation
 
-This guide will help you set up an example etcd cluster that you can use with CN-WAN Operator. For the scope of this demo and keep things simple, the etcd cluster will consist of only one node - or instance.
+**IMPORTANT NOTE**: this guide will only help you create a **demo** cluster so that you can quickly have a working example to use with the CN-WAN Operator and is not intended to be used in production. We strongly encourage you to follow more thorough guides if you want to use etcd in production.
 
-In the first part, *Create the etcd cluster*, we will install and start etcd. The second part, *Make it more secure*, is not mandatory but **highly** suggested.
+The first section focuses on installing etcd in the same Kubernetes cluster as the operator. For a standalone installation jump to *Create the etcd cluster*. The section *Make it more secure*, is not mandatory but **highly** suggested.
 
-**IMPORTANT NOTE**: this guide will only help you create a **demo** cluster so that you can quickly have a working example to use with the CN-WAN Operator and is not intended to be used in production. We strongly encourage you to follow more thorough guides if you want to use etcd in production.
+## Quick install on Kubernetes
+
+Since the operator runs on Kubernetes, it's safe to assume you have a Kubernetes cluster available, and the simplest way to get up and running quickly with etcd is to install it on the same Kubernetes cluster where you will run the operator. Please note that this etcd installation is separate from the etcd service that comes bundled with Kubernetes.
+
+If you prefer to use a binary on your favorite OS or docker instead, jump to the next section.
+
+We will use *Helm* to install etcd on Kubernetes. The `bitnami` Helm repo holds a chart that is well maintained, so let's add that repo:
+
+    helm repo add bitnami https://charts.bitnami.com/bitnami
+    helm repo update
+
+Check customization options for the `bitnami/etcd` chart:
+
+    helm show values bitnami/etcd | less
+
+Custom options can be added to a `values.yaml` file, or specified on the command line. To install etcd on a Kubernetes cluster in a public cloud using a load balancer and authentication enabled, run this:
+
```

Comment:
I think it would be better to explain what the following command does here. For example: "we're going to create the pod, expose it with .., create the RBAC and create a root user with password `dem0-pwd`.

Not really a super important point, but I think it would give context to it